### PR TITLE
Added registration phase and users permissions

### PR DESCRIPTION
**Pull request by cassandradamico**

### CODE REVIEW

File: mkdocs.yml [hunk 1]
```diff
@@ -50,8 +51,25 @@ nav:
...
     - 'Program Team': 
       - 'Team page': 'general/team-page.md'
       - 'Add team members': 'general/add-team-members.md'
       - 'Edit team members roles': 'general/edit-team-roles-program.md'
       - 'Remove team members': 'general/remove-team-members-program.md'
+    - 'Program Design':
+      - 'Program Design': 'ProgramOverview/design.md'
+      - 'Read & Change Program Design': 'ProgramOverview/read-change-design-details.md'
+    - 'Registration':
+      - 'Registration page': 'ProgramOverview/registration.md'
```
Comment (tijsziere) [inline mkdocs.yml:66]: I would not call it 'Registration page', Maybe 'Registrations overview'?

File: mkdocs.yml [hunk 2]
```diff
@@ -50,8 +51,25 @@ nav:
...
+      - 'Import registration list with excel': 'ProgramOverview/registration-import-list.md'
```
Comment (tijsziere) [inline mkdocs.yml:67]: I would call this 'Import excel file'

File: mkdocs.yml [hunk 3]
```diff
@@ -50,8 +51,25 @@ nav:
...
+      - 'Automatic import with KOBO': 'ProgramOverview/registration-automatic-import-kobo.md'
+      - 'Deduplication': 'ProgramOverview/registration-deduplication.md'
+      - 'Validate registration': 'ProgramOverview/registration-validate-pa.md'
+      - 'Reject registration': 'ProgramOverview/registration-validate-pa.md'
+      - 'Empty Registration List': 'ProgramOverview/registration-list-empty.md'
```
Comment (tijsziere) [inline mkdocs.yml:72]: I would remove this as a seperate item and make the content available on the registration.md page

File: docs/ProgramOverview/registration-list-empty.md
Comment (tijsziere) [inline docs/ProgramOverview/registration-list-empty.md:1]: I would remove this file and make the contents part of the registrations.md page

File: docs/ProgramOverview/registration-import-list.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+Unless you automatically import registrations with KOBO form or equivalent, a registration list can be manually imported with an excel file (.csv format) in the program. 
+Enter the Program which you want to import your registration file. Then, go to Registration page. You can now start importing your file.
+
+!!! info "Automatic Import of registration"
+    KOBO Toolbox App is integrated with 121 platform, enabling automatic import of new registrations of beneficiaries. Follow the instructions on this page.
+    Want to integrate your KOBO form or alternative registration App with 121 platform ? Please, contact our team.
+
+### **Import a registration list**
+
+[Import Registration button](../assets/img/RegistrationImportFile.png)
```
Comment (tijsziere) [inline docs/ProgramOverview/registration-import-list.md:18]: this needs an '!' in front of it to display the image

File: docs/ProgramOverview/registration-import-list.md [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+
+Click on Import Registrations button, using a CSV file.​
+Import registration pop-up.
+
+[Import Registration pop-up](../assets/img/RegistrationImportFileCSVBox.png)
```
Comment (tijsziere) [inline docs/ProgramOverview/registration-import-list.md:23]: same here

File: docs/ProgramOverview/registration-import-list.md [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+!!! warning "Unsucessful import"
+    In case the import is unsuccessful or show an error message, kindly check your excel file is correctly formatted and saved as .CSV file.
+    Should you need further guidance or support, Please do not hesitate to contact our <a href="mailto:support@121.global">support team</a>.
+
+### **Download template registration file**
+
+The template is automatically generated in 121 platform, based on the program registration form.
+For instance, the program registration form may include selection criteria such as name, phone number, amount of household members, province and IBAN number. The template will automtically include a column per criteria, matching the registration form questions. Below, you can find an example of a template based on a KOBO registration form.
+
+INCLUDE 2 PICTURES MATCHING 5 CRITERIA + COLUMNS
```
Comment (tijsziere) [inline docs/ProgramOverview/registration-import-list.md:39]: You can put comments in your markdown file by doing
`<!-- your text -->`

File: docs/ProgramOverview/design.md [hunk 7]
```diff
@@ -3,14 +3,36 @@ title: Program overview - design
...
-<div class="notice notice-error">
-Currently it is not possible to change these metrics yourself. Please <a href="mailto:support@121.global">contact the 121 support team</a> to help you with this.
-</div>
+![Program Details Overview](../assets/img/ProgramOverview.png)
+
+### Program Details setup metrics
+
+The Program Details metrics translates your program requirements. The details provide the main information about the duration and location of your program, budget, targeted beneficiaries, cash aid delivery and distribution frequencies. Each of your active programs may have different metrics shown in the table below.
+
+Futher explanation about the program details metrics on the [Program Details page.](../ProgramOverview/design-details.md)
```
Comment (tijsziere) [inline docs/ProgramOverview/design.md:22]: This link does not work for me

File: mkdocs.yml
Comment (tijsziere) [inline mkdocs.yml:1]: consider removing everything under and including 'Running Your Program'

File: docs/ProgramOverview/registration-reject-pa.md [hunk 8]
```diff
@@ -0,0 +1,41 @@
...
+- A pop-up appears to confirm your action **You are about to Mark as declined for X People Affected.**
+- Click on **OK**. A message indicates the completion of the action.
+
+Rejected People Affected remain listed in the Registration & Validation page with the status **Rejected**. Filter the list by status to have an overview of all rejected PA in this program.
+
+### **Validation and deduplication process**
+
+Your validation process may include :
+
+- Deduplication of People Affected in your registration list. Follow the instructions [on the deduplication page](../programoverview/registration-deduplication.md)
```
Comment (tijsziere) [inline docs/ProgramOverview/registration-reject-pa.md:28]: This link does not work

**Review by tijsziere — CHANGES_REQUESTED** [review]

File: mkdocs.yml (hunk 1 above)
Comment (cassandradamico) [reply mkdocs.yml:66]: Yes, that's fine to me. to be more consistent, I will do the same for the other page, as Team page = Team Overview ? etc.

File: mkdocs.yml (hunk 3 above)
Comment (cassandradamico) [reply mkdocs.yml:72]: @tijsziere you mean all ? 
I would keep each action separate. The empty list you mean ? then this I can make it available as a ''good to know'' on the registration page

**Pull request merged**